server/src/code-runner/downloader/docker-file-downloader.ts

```diff
@@ -1,7 +1,4 @@
-import { Observable } from 'rxjs/Observable';
-import { empty } from 'rxjs/observable/empty';
-import { merge } from 'rxjs/observable/merge';
-import { concat as concatObservable } from 'rxjs/observable/concat';
+import { Observable, EMPTY, merge, concat as concatObservable } from 'rxjs';
```

Comment:
Why the alias for `concat`?

Reply:
That's what rxjs-tslint does

Reply:
Actually, it's not what rxjs-tslint does. The name we had before.

Reply:
Because below we import `concat` from the operators

Reply:
Ah, so it's rxjs-tslint's way of distinguishing between those two. That makes sense.